src/model/page.urls.ts

```diff
@@ -1,5 +1,8 @@
 export const ROOT: string = "/";
+export const PTF: string = "/promise-to-file";
 export const PROMISE_TO_FILE: string = "/promise-to-file";
 export const COMPANY_NUMBER: string = "/company-number";
+export const CHECK_COMPANY: string = "/check-company";
+export const PTF_CHECK_COMPANY: string = PTF + CHECK_COMPANY;
```

Comment:
We should standardise on shortening 'promise to file' to 'ptf' in variable/constant names if this is preferable (so refactor the constant name below at this point too).

Reply:
Using the abbeviation "PTF" for long names but for the URL itself we use dthe full name "PROMISE_TO_FILE" this way we get to be as brief as possible here we need to be and as descriptiive as possible where can be.

Reply:
Makes sense.